core/templates/components/question-directives/question-misconception-editor/question-misconception-editor.component.ts

```diff
@@ -218,6 +224,25 @@ export class QuestionMisconceptionEditorComponent implements OnInit {
     this.misconceptionEditorIsOpen = true;
   }
 
+  removeMisconception(): void {
+    this.selectedMisconception = null;
+    this.selectedMisconceptionSkillId = null;
+    this.feedbackIsUsed = false;
+    this.misconceptionName = '';
```

Comment:
Don't use empty string as a sentinel value for null. Ditto for emptyTaggedMisconception.skillId below.